python/py-rabbyt/Portfile

```diff
@@ -16,22 +16,18 @@ long_description    Rabbyt is a sprite library for Python with game \
 
 platforms           darwin
 
-homepage            http://matthewmarshall.org/projects/rabbyt/
+homepage            http://arcticpaint.com/projects/rabbyt/
 master_sites        pypi:R/Rabbyt/
 distname            Rabbyt-${version}
 use_zip             yes
 
-checksums           md5     bb603a5cb02f3a8bb2e674f60e62de2c \
-                    sha1    decb2b398b90592b94a509520eb11645acb132b5 \
-                    rmd160  c5affa8472ecde15f3c59860332f5dae9d09ac6a
+checksums           rmd160  c5affa8472ecde15f3c59860332f5dae9d09ac6a \
+                    sha256  23ee10879f65be1bd2e916380df5531917d042dedf5cc0b573976ab4090a61f6 \
+                    size    171158
 
-python.versions     26
+python.versions     27
 
 if {${name} ne ${subport}} {
     depends_build       port:py${python.version}-setuptools
     livecheck.type      none
-} else {
-    livecheck.type      regex
-    livecheck.url       https://pypi.python.org/pypi/rabbyt/json
-    livecheck.regex     "Rabbyt-(\\d+(?:\\.\\d+)*)"
```

Comment:
Any reason to drop livechecking?

Reply:
I am not dropping the livecheck, just removing redundant lines. When using pypi as master_site, the livecheck.type is automatically set to pypi.
```DEBUG: Loading the defaults from '/Users/renee/Software/macports-ports/_resources/port1.0/livecheck/pypi.tcl'```